packages/venia-concept/src/components/TextArea/__tests__/textArea.spec.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import { Form } from 'informed';
+import TextArea from '../textArea';
+
+jest.mock('src/classify');
+
+const classes = {
+    input: {}
+};
```

Comment:
I wouldn't pass classes in. It's better to just mock `classify` (which you're doing) and use the default classes, I think.